packages/plexis/src/index.js

```diff
@@ -1,6 +1,7 @@
 export {default as toPred, default as toPredecessor} from '@plexis/to-pred';
 export {default as toSucc, default as toSuccessor} from '@plexis/to-succ';
 export {default as toTitle, default as titleize} from '@plexis/to-title';
+export {default as toLower} from '@plexis/to-lower';
```

Comment:
Can you please create the appropriate aliases as described in the corresponding issue https://github.com/plexis-js/plexis/issues/6?